api/include/opentelemetry/nostd/variant.h

```diff
@@ -1,1281 +1,33 @@
-// MPark.Variant
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
Moved contents of MPark.Variant to `mpark/variant.h` and replaced with shim on top of Standard Library and Abseil.